packages/web/src/pages/artist-coins-launchpad-page/components/ArtistCoinsTable.tsx

```diff
@@ -415,9 +408,7 @@ export const ArtistCoinsTable = ({ searchQuery }: ArtistCoinsTableProps) => {
     (e: React.MouseEvent<HTMLTableRowElement>, rowInfo: any) => {
       const coin = rowInfo.original
       if (coin?.ticker) {
-        navigate(
-          ASSET_DETAIL_PAGE.replace(':ticker', formatTickerForUrl(coin.ticker))
-        )
+        navigate(ASSET_DETAIL_PAGE.replace(':ticker', coin.ticker))
```

Comment:
are the tickers not going to have the $ in front at all now? or just in the urls?